File: backend/java/movie_service/src/main/java/com/ticketflow/movie_service/models/client_models/AddMovieRequestModel.java
package com.ticketflow.movie_service.models.client_models;

import java.time.LocalDateTime;

public class AddMovieRequestModel {
    private LocalDateTime startTime;
    private Integer filmId;
    private Integer cinemaHallId;

    public AddMovieRequestModel(LocalDateTime startTime, Integer filmId, Integer cinemaHallId) {
        this.startTime = startTime;
        this.filmId = filmId;
        this.cinemaHallId = cinemaHallId;
    }

    public LocalDateTime getStartTime() {
        return startTime;
    }

    public Integer getFilmId() {
        return filmId;
    }

    public Integer getCinemaHallId() {
        return cinemaHallId;
    }
}

File: backend/dotnet/ticketflow/TicketFlow.MovieService/WebApi/ClientModels/Requests/AddMovieRequestModel.cs
using System;

namespace TicketFlow.MovieService.WebApi.ClientModels.Requests
{
    public class AddMovieRequestModel
    {
        public DateTime StartTime { get; set; }

        public int FilmId { get; set; }

        public int CinemaHallId { get; set; }
    }
}